cmd/kube-gateway/app/proxy.go

```diff
@@ -132,3 +140,16 @@ func buildProxyHandlerChainFunc(clusterManager clusters.Manager, enableAccessLog
 func GetNativeOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return nativeopenapi.GetOpenAPIDefinitions(ref)
 }
+
+// proxyHTTPErrorLogWriter serves as a bridge between the standard log package and the klog package.
+// It also filter out some noisy http error log
+type proxyHTTPErrorLogWriter struct{}
+
+// Write implements the io.Writer interface.
+func (writer proxyHTTPErrorLogWriter) Write(data []byte) (n int, err error) {
+	if bytes.HasPrefix(data, []byte("http: TLS handshake error from")) {
```

Comment:
will it have effect if data is cut off into segments?

Reply:
this http error mesage is written all at once from http pakcage, will not be cut into segments

Reply:
get it